### ISSUE TITLE
Yamaha4OP patchbank loading broken in V50/YS200 mode

### ISSUE THREAD

**Issue by mtarenskeen** [eclab/edisyn#68]
Hi, maybe you are preparing changes to support V50/TQ5/YS200 100 patch/bank librarian format. But right now in current GIT version both 32-patch banks and 100-patch banks give error messages when trying to load them in the librarian while in V50 or YS/TQ mode.  Work in progress?

**Comment by Jodeo87** [comment]
Following this and hoping for an update; I have a Yamaha V50. Using the Multimode Editor, I'm able to only load 24 Performances (of 100) into the Librarian.

**Comment by mtarenskeen** [comment]
> Following this and hoping for an update; I have a Yamaha V50. Using the Multimode Editor, I'm able to only load 24 Performances (of 100) into the Librarian.

Current version (35) has improved support for V50 Single Mode 100 patch/banks (Thanks to Sean). Not (yet) for 100 performance/banks. Did you get my e-mail response? My e-mail client/provider is doing strange lately and my mails often end up in SPAM boxes. You might want to check if you did not receive anything from me.

**Comment by Jodeo87** [comment]
>> Did you get my e-mail response? My e-mail client/provider is doing strange lately and my mails often end up in SPAM boxes. You might want to check if you did not receive anything from me.

Yes, thank you. I really appreciated all you shared in your email! Thank you so much for making time to reply.
